docs/source/azure/policy/resources/keyvault.rst

```diff
@@ -73,19 +74,26 @@ This policy will find all KeyVaults with 10 or less API Hits over the last 72 ho
               - type: asq
                 queue: https://accountname.queue.core.windows.net/queuename
 
-This policy will find all KeyVaults with an access of Service Principals not in the white list
+This policy will find all KeyVaults with an access of Service Principals not in the white list that exceed read-only access
 
 .. code-block:: yaml
 
     policies:
         - name: policy
           description:
-            Adds a tag to all virtual machines
+            Ensure only authorized people have an access
           resource: azure.keyvault
           filters:
-            - type: whitelist
-              key: accessPolicies[].principalName
-              op: difference
-              value:
-                - account1@sample.com
-                - account2@sample.com
+            - not:
+              - type: whitelist
+                key: principalName
+                users:
+                  - account1@sample.com
+                  - account2@sample.com
+                permissions:
+                  keys:
+                    - get
```

Comment:
What are the other options other than get? Maybe add those in the policy description.

Reply:
Done